multiples.rb

```diff
@@ -0,0 +1,17 @@
+class Multiples
+  def self.multiples(limit)
+
+    multiples = []
+
+    limit.times do |n|
+        if n % 3 == 0
+            multiples.push(n)
+        elsif n % 5 == 0
+            multiples.push(n)
```

Comment:
Good. Also consider using boolean operators to avoid the repetition of the push statement